metadata_registry/grpc/server.py

```diff
@@ -0,0 +1,335 @@
+from concurrent import futures
+import grpc
+import logging
+import sqlalchemy as db
+from sqlalchemy.orm import Session
+import pymysql
+
+import ensembl_metadata_pb2
+import ensembl_metadata_pb2_grpc
+from config import MetadataRegistryConfig
+
+pymysql.install_as_MySQLdb()
+config = MetadataRegistryConfig()
+
+
+def load_database(uri=None):
+    if uri is None:
+        uri = config.METADATA_URI
+    engine = db.create_engine(uri)
+    connection = engine.connect()
+    if not connection:
+        raise ValueError(f'Could not connect to database {uri}.')
+
+    connection.close()
+    return engine
```

Comment:
`engine.connect()` will raise `OperationalError` if it can't connect so I'd do:

```suggestion
    try:
        connection = engine.connect()
    except db.exc.OperationalError as err:
        raise ValueError(f'Could not connect to database {uri}: {err}.') from err

    connection.close()
    return engine
```
Although if for some reason the DB becomes unreachable after `load_database()` has been called, then `OperationalError` exceptions must be handled in other functions that execute queries.

Reply:
Better to ask for forgiveness than permission!